README.md

```diff
@@ -23,7 +23,7 @@ To write _SQML_ code, the following is required:
 - [Eclipse IDE for Java Developers](https://www.eclipse.org/downloads/packages/), e.g. 2020-09 (4.17) or higher
 - _SQML Eclipse Plugin_
 
-:warning: There is currently no publicly available update site to download the _SQML Eclipse Plugin_ from. This will change when our migration to _Github_ is completed. Until then, an organization has to build and publish the plugin internally.
+The _SQML Eclipse Plugin_ can be installed from our [Eclipse update site](https://softicar.github.io/sqml/).
```

Comment:
This link leads to the following page:
![image](https://user-images.githubusercontent.com/15086658/141147851-a21c8ac6-abc4-438d-8f10-414a302b9efe.png)
Is this indirection intended?

Reply:
it is, please try again, that page was updated